packages/fx-core/resource/package.nls.json

```diff
@@ -136,6 +136,7 @@
   "core.migrationV3.VS.Message": "The Teams Toolkit preview feature - Teams App Configuration Improvements is turned on. This project is required to be upgraded to work with the preview features. Select \"Learn more\" to read about the required changes or select \"Upgrade\" to perform the changes automatically. If you don't want to upgrade, you can turn off the preview feature.",
   "core.migrationV3.CreateNewProject": "Teams toolkit's pre-released version supports new project configuration and is incompatible with previous versions. Please try it by creating a new project, auto upgrade support will be added in the coming release.",
   "core.migrationV3.abandonedProject": "This project is only for previewing and will not be supported by Teams Toolkit. Please try Teams Toolkit by creating a new project",
+  "core.migrationV3.aadManifestNotExist": "templates/appPackage/aad.manifest.template.json does not exist. You may trying to migrate a project created by Teams Toolkit < 3.8.0. Please try to install Teams Toolkit with Version 4.2.2 and run migration first.",
```

Comment:
"You may trying to migrate a project created by Teams Toolkit < 3.8.0. Please try to install Teams Toolkit with Version 4.2.2 and run migration first."  --> "You may be trying to migrate a project created by Teams Toolkit < v3.8.0. Please install Teams Toolkit v4.2.2 and run migration first."

Reply:
Updated, thx!